apps/vectordb/tests/test_vectorstore.py

```diff
@@ -0,0 +1,143 @@
+"""Test PGVector functionality.
+
+Ported from https://github.com/hwchase17/langchain/blob/master/tests/integration_tests/vectorstores/test_pgvector.py
+"""
+
+import pytest
+from langchain.docstore.document import Document
+from langchain_community.embeddings import DeterministicFakeEmbedding
+
+from apps.utils.factories.experiment import ExperimentFactory
+from apps.vectordb.const import META_ALL
+from apps.vectordb.models import ADA_TOKEN_COUNT
+from apps.vectordb.vectorstore import DistanceStrategy, PGVector
+
+
+@pytest.fixture()
+def experiment(db):
+    return ExperimentFactory()
+
+
+@pytest.mark.django_db()
+def test_vectorstore(experiment):
+    """Test end to end construction and search."""
+    texts = ["foo", "bar", "baz"]
+    docsearch = PGVector.from_texts(
+        texts=texts,
+        embedding=DeterministicFakeEmbedding(size=ADA_TOKEN_COUNT),
+        experiment=experiment,
+    )
+    output = docsearch.similarity_search("foo", k=1)
+    _remove_meta_fields(output)
```

Comment:
Why's the meta fields removed here?

Reply:
yes, just for testing so that we can compare the results without having to compare the metadata